clients/tv/src/serviceWorker.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable */
```

Comment:
Ten service worker to chyba w ogóle to wyrzucenia jest co? Przecież nie będziemy nic z SW robić, a w przyszłości jak będziemy chcieli to dodamy. Trochę pachnie na dead code.

Reply:
wywaliłem